src/openforms/formio/tests/test_validation.py

```diff
@@ -0,0 +1,31 @@
+from django.test import SimpleTestCase
+
+from rest_framework.serializers import ValidationError
+
+from ..validation import validate_formio_data
+
+
+class FormioValidationTests(SimpleTestCase):
+    def test_textfield_required_validation(self):
```

Comment:
I think that codecov also wants a test where there is no `validate` in the component